src/type.rs

```diff
@@ -98,65 +55,43 @@ impl Display for Type {
     }
 }
 
+impl From<SrType> for Type {
+    fn from(mut type_definition: SrType) -> Self {
+        match type_definition.main_type.as_str() {
+            "Int32" => Type::Int32,
+            "Int64" => Type::Int64,
+            "Int128" => Type::Int128,
+            "Int256" => Type::Int256,
+            "Uint32" => Type::Uint32,
+            "Uint64" => Type::Uint64,
+            "Uint128" => Type::Uint128,
+            "Uint256" => Type::Uint256,
+            "String" => Type::String,
+            "ByStr20" => Type::ByStr(20),
```

Comment:
Maybe this one should be moved to the generic case and identified with a regex instead?

Reply:
Although I don't need them for now, I created #5 ticket for them.